gax/src/main/java/com/google/api/gax/core/GoogleCredentialsProvider.java

```diff
@@ -62,6 +67,16 @@
 
   @Override
   public Credentials getCredentials() throws IOException {
+    return getCredentials(false, null);
+  }
+
+  /**
+   * If user doesn't provide custom endpoint and scopes, and the credentials is service account
```

Comment:
is --> are

Reply:
Thank you for all the comments. Will address them in the next commit.